sql/sql_udf.cc

```diff
@@ -206,15 +206,6 @@ void udf_init()
     DBUG_PRINT("info",("init udf record"));
     LEX_CSTRING name;
     name.str=get_field(&mem, table->field[0]);
-
```

Comment:
Here we have older commit which we don't need.
You will have to do git rebase and push new changes.